package.json

```diff
@@ -29,15 +29,15 @@
     "version": "npm run clean && npm run build"
   },
   "peerDependencies": {
-    "playwright": "*"
+    "playwright": "~0.12.1"
```

Comment:
```suggestion
    "playwright": "^0.12.1"
```

Reply:
I normally do but playwright released breaking changes from 0.11.0 -> 0.12.0 and they could do the same for 0.13.0. That is why I did `~` instead of `^`.